docs/configuration/configuration.md

```diff
@@ -163,9 +164,14 @@ no operator or function is defined for this character.
 ### Zone Id
 
 The time zone id. By default, the system default zone ID is used.
+
 ```java
 ExpressionConfiguration configuration=ExpressionConfiguration.builder()
         .zoneId(ZoneId.of("Europe/Berlin"))
         .build();
 ```
 
+### Single Quote String Literals
```

Comment:
move this chapter above the "Strip Trailing Zeroes" chapter, the settings are sorted alphabetically.